src/preset.ts

```diff
@@ -1657,3 +1657,11 @@ export function configurePresetChangeNeedsClean(newPreset: ConfigurePreset, oldP
         return false;
     }
 }
+
+export function getValueStrategy(value: string | ValueStrategy): string | undefined {
```

Comment:
I don't think `strategy` should be in the name.  You don't get the strategy, you get the value.